ui/components/PageLayout.tsx

```diff
@@ -27,6 +27,7 @@ const PageLayout: FC<Props> = ({ children, title, user, isSelfServe }) => {
           href="https://assets.ubuntu.com/v1/16c27f81-COF%20favicon-16x16.png"
         />
         <title>{title}</title>
+        <script src="../.well-known/webauthn.js" />
```

Comment:
I think this needs to be dynamic, perhaps we can fix this as part of https://github.com/canonical/identity-platform-login-ui/issues/305